build.gradle

```diff
@@ -819,6 +857,12 @@ project(':aeron-system-tests') {
     }
 }
 
+project(':aeron-test-support') {
+    dependencies {
+        implementation "org.junit.jupiter:junit-jupiter-api:${junitVersion}"
```

Comment:
The dependency on JUnit should be `api` instead of `implementation` since this module depends in it's public API on the JUnit classes.